### DISCUSSION TITLE
Cold hands, warm ... what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#342] [Galaxy Trivia] [answered]
**Hint:** ♥️

**Comment by GameExplorer** [comment]
Heart?

**Comment by thuongtruong1009** [comment, marked as answer]
The saying is "Cold hands, warm heart." This phrase means that a person who appears cold or aloof on the surface may actually have a warm and caring personality underneath. It suggests that external appearances can be deceiving and that we should not judge someone solely based on their outward demeanor.

**Comment by peterbaccaro** [comment]
The phrase "cold hands, warm heart" is a saying that suggests that people with cold hands may have a warm and generous personality. The saying is often used in a figurative sense, to suggest that someone who seems reserved or distant may actually have a kind and caring nature.
